info/machine.go

```diff
@@ -17,9 +17,10 @@ package info
 type FsInfo struct {
 	// Block device associated with the filesystem.
 	Device string `json:"device"`
-
 	// Total number of bytes available on the filesystem.
 	Capacity uint64 `json:"capacity"`
+	// Free number of bytes available on the filesystem.
+	Free uint64 `json:"free"`
```

Comment:
This should probably go into "root" cgroup stats. I am curious as to why this is necessary since you can compute 'free' from 'capacity' which is here and 'usage' which is in "root" cgroup stats.